src/main/webapp/play.html

```diff
@@ -275,14 +277,12 @@
 		// If it's not dash, play with videojs
 		if(extension != dashExtension){
 
-			var player = videojs('video-player', {
```

Comment:
poster deleted because the settings don't change after video initialized and at 287 poster also initialized again.